integration-tests/src/main/scala/io/stryker4s/RoboBarServer.scala

```diff
@@ -0,0 +1,39 @@
+package io.stryker4s
+
+import cats.effect._
+import cats.implicits._
+import io.stryker4s.endpoint.{DrinksEndpoint, HealthCheckEndpoint, OrderEndpoint}
+import io.stryker4s.repository.{DrinksRepository, OrderRepository}
+import io.stryker4s.service.{DrinksService, OrderService}
+import io.stryker4s.validator.OrderValidator
+import org.http4s.implicits._
+import org.http4s.server.blaze._
+import org.http4s.server.{Router, Server}
+
+import scala.language.higherKinds
+
+object RoboBarServer extends IOApp {
+
+  private[this] val root = "/stryker4s"
```

Comment:
Shouldn't this be robobar?
```suggestion
  private[this] val root = "/robobar"
```